batch/batch/driver/scheduler.py

```diff
@@ -206,19 +217,19 @@ def allocate_cores(user, mark):
                     instance.adjust_free_cores_in_memory(-record['cores_mcpu'])
                     should_wait = False
                     scheduled_cores_mcpu += record['cores_mcpu']
-                    to_schedule.append((record, instance))
-
-        results = await bounded_gather(*[functools.partial(schedule_job, self.app, record, instance)
-                                         for record, instance in to_schedule],
-                                       parallelism=100,
-                                       return_exceptions=True)
-
-        for ((record, instance), result) in zip(to_schedule, results):
-            batch_id = record['batch_id']
-            job_id = record['job_id']
-            id = (batch_id, job_id)
-
-            if isinstance(result, Exception):
-                log.info(f'error while scheduling job {id} on {instance}, {result!r}')
 
+                    async def with_error_handling(id, instance, f):
```

Comment:
Can you adding schedule to the name of the function? Trying to avoid accidental mixups since they have the same name right now.